src/applications/check-in/components/layout/Wrapper.jsx

```diff
@@ -22,6 +30,11 @@ const Wrapper = props => {
     ? 'vads-u-padding-y--2'
     : ' vads-u-padding-y--3';
 
+  const selectApp = useMemo(makeSelectApp, []);
+  const { app } = useSelector(selectApp);
+  const downtimeDependency =
+    app === 'dayOf' ? externalServices.cie : externalServices.pcie;
```

Comment:
```suggestion
    app === APP_NAMES.CHECK_IN ? externalServices.cie : externalServices.pcie;
```

should it be this for consistency?

Reply:
oh yep, thanks for catching that!